apps/website/src/content/docs/notificationmarker.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: NotificationMarker
+description: Shows a notification marker dot at the top right of the component.
+propsPath: '@itwin/itwinui-react/esm/core/NotificationMarker/NotificationMarker.d.ts'
+thumbnail:
+---
+
+<p>{frontmatter.description}</p>
+
+<LiveExample src='NotificationMarker.main.jsx'>
+  <AllExamples.NotificationMarkerMainExample client:load />
+</LiveExample>
+
+## Usage
+
+### Status
+
+`NotificationMarker`s can have `primary` (default), `positive`, `warning`, `negative`, or `white` statuses. The `white` status is used on colored backgrounds, such as when using it in the [high visibility](button#high-visibility) or the [call-to-action](button#call-to-action) buttons.
+
+<LiveExample src='NotificationMarker.statuses.jsx'>
+  <AllExamples.NotificationMarkerStatusesExample client:load />
+</LiveExample>
+
+### Controlled
+
+The `enabled` prop can be used to programatically show or hide the notification marker.
```

Comment:
This section is wrong and misleading.

- "Controlled" refers to when the consumer needs to maintain the component state (doesn't apply here because there is no state).
- "Programmatically" refers to when a consumer calls a function, such as `.show()`.

Reply:
Removed the "Controlled" section and merged its contents with the "Usage" section (https://github.com/iTwin/iTwinUI/pull/1999/commits/370c4788876dc28a7ad2f182e9e0b072c7945e62).

Reply:
Where are you getting this idea of "to prevent layout shifts"? The `enabled` prop has nothing to do with layout shifts.

Reply:
If the user's styles are in such a way that they don't handle the notification marker wrapper correctly, I felt that there could be some layout shifting that way.

`enabled` can be used to always keep the wrapper in the DOM instead of conditionally rendering it.

```jsx
{notifications.length > 0 ? <NotificationMarker>{content}</NotificationMarker> : content}
```

Open to better ways to phrase this. I could even remove the mention of `enable` if we feel that it can be inferred by users.

Reply:
I see what you mean. In that case, I think we should focus on what exactly it does and when the user might want to use it. It can be documented in a new subheading with a short, static example.

Here's my suggestion:

> ### Conditional rendering
> 
> There are many situations when the notification dot needs to appear/disappear depending on the application lifetime. Instead of conditionally rendering the `NotificationMarker` in such situations, the `enabled` prop can be used.
> 
> ```jsx
> <NotificationMarker enabled={notifications.length > 0}>…</NotificationMarker>
> ```
> 
> When `enabled` is set to `false`, the DOM element will still be present, but the notification marker will not be displayed visually.

Reply:
Thanks, done (https://github.com/iTwin/iTwinUI/pull/1999/commits/a8a2909e1d5ac11880fca626fc358156aec19fb4).